src/components/ModelCamera/components/Model.ts

```diff
@@ -0,0 +1,75 @@
+import * as tf from "@tensorflow/tfjs";
+import { Models } from "../../../types/Models";
+import { isMobile } from 'react-device-detect';
+import * as tflite from '@tensorflow/tfjs-tflite'
+// Adds the CPU backend.
+//import '@tensorflow/tfjs-backend-cpu';
+export async function loadModel() {
+    let result: Models = { R_Model: undefined, L_Model: undefined };
+
+    if (isMobile) {
+        tflite.setWasmPath(
+            'https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-tflite@0.0.1-alpha.8/dist/'
+        );
+        console.log('isMobile');
+        result.R_Model = await tflite?.loadTFLiteModel('https://tflite-asl-models.s3.amazonaws.com/R_converted_model.tflite')
+        result.L_Model = await tflite?.loadTFLiteModel('https://tflite-asl-models.s3.amazonaws.com/L_converted_model.tflite')
+    } else {
+        console.log("isDesktop");
+        result.R_Model = await tf?.loadGraphModel(
+            "https://signy-asl-models.s3.amazonaws.com/models/R_graph_model/model.json"
+        );
+        result.L_Model = await tf?.loadGraphModel(
+            "https://signy-asl-models.s3.amazonaws.com/models/L_graph_model/model.json"
+        );
+    }
+
+
+
+
+
+    return result;
+}
+
+export function calc_landmark_list(
```

Comment:
We should probably try to stick with one casing strategy. Either snake or camel. We should probably go with camel case.

Reply:
Also, when is this function used?

Reply:
This function is for preprocessing the landmarks of the mediapipe hands solution output